src/manager.cpp

```diff
@@ -219,9 +228,9 @@ sdbusplus::async::task<>
                 {
                     break;
                 }
-                for ([[maybe_unused]] const auto& dataOp : dataOperations)
+                for (const auto& dataOp : dataOperations)
                 {
-                    co_await syncData(dataSyncCfg);
+                    co_await syncData(dataSyncCfg, std::get<0>(dataOp));
```

Comment:
Actually, we no need to use `get` API, just use `dataOp.first`.

also, I don't think we need to use `std::map`..could be `std::vector`